uploader/pantheon.py

```diff
@@ -283,6 +287,18 @@ def process_file(path, filetype):
             if not args.dry:
                 r = requests.post(url, headers=HEADERS, files=files, auth=(args.user, pw))
                 _print_response('resource', path, r.status_code, r.reason)
+    elif isAssembly:
+        url += '/' + path.name
+        logger.debug('url: %s', url)
+        jcr_primary_type = 'pant:assembly'
+        data = _generate_data(jcr_primary_type, base_name, path.name, asccidoc_type='nt:file')
+        # This is needed to add a new module version, otherwise it won't be handled
+        data[':operation'] = 'pant:newDraftVersion'
```

Comment:
Pointing out here that if you change the servlet operation to 'pant:newAssemblyVersion' as Carlos suggested, you'd need to change it here, too.